ssf/files/tofs_.github/docs/CONTRIBUTING.rst

```diff
@@ -55,6 +55,22 @@ So based on the example above:
 * The ``(scope):`` will be shown in bold text without the brackets.
 * The ``subject`` follows the ``scope`` as standard text.
 
+pre-commit
+^^^^^^^^^^
+
+`pre-commit <https://pre-commit.com/>`_ is configured for this formula, which you may
```

Comment:
So one example is here: "is configured for this formula" -- but this text will only be seen in the `.github` repo (while it is linked to from each formula).

Reply:
Happy to change, but I was following the pattern in the rest of the page - several other uses of "this formula".

Reply:
Fair enough, let's figure this out later.

Reply:
Pushed to `.github` repo: https://github.com/saltstack-formulas/.github/commit/0a7e31715526cf138fda65404fefaccf3a8c106e.